src/components/reducers/index.tsx

```diff
@@ -0,0 +1,16 @@
+import { SUBMIT } from "../types";
+
+export const figma = (state = { user_data: [] }, action: any) => {
+  switch (action.type) {
+    case SUBMIT: {
+      return {
+        ...state,
+        user_data: [...state.user_data, action.payload],
```

Comment:
i think you should be using it like this
user_data: {...state.user_data, action.payload}
this will create data like this
{
// user data
}